### PR TITLE
move ListFooterAction to interface file

### PR DESCRIPTION
**Pull request by omrihecht**

### CODE REVIEW

File: projects/ui-framework/src/lib/form-elements/lists/list-footer/list-footer.component.ts [hunk 1]
```diff
@@ -1,12 +1,7 @@
 import { Component, EventEmitter, Input, Output } from '@angular/core';
 import { ButtonSize, ButtonType } from '../../../buttons-indicators/buttons/buttons.enum';
 import { Icons } from '../../../icons/icons.enum';
-
-export interface ListFooterActions {
-  clear?: true;
-  cancel?: true;
-  apply?: true;
-}
+import { ListFooterActions } from './list-footer.interface';
```
Comment (galloween) [inline projects/ui-framework/src/lib/form-elements/lists/list-footer/list-footer.component.ts]: why not put it in the list.interface.ts file?

File: projects/ui-framework/src/lib/form-elements/lists/list-footer/list-footer.interface.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export interface ListFooterActions {
+  clear?: true;
```
Comment (galloween) [inline projects/ui-framework/src/lib/form-elements/lists/list-footer/list-footer.interface.ts]: true is not a type...
Am I missing something?

Comment (omrihecht) [reply projects/ui-framework/src/lib/form-elements/lists/list-footer/list-footer.interface.ts]: what do you mean by type?
it's an interface

Comment (omrihecht) [reply projects/ui-framework/src/lib/form-elements/lists/list-footer/list-footer.interface.ts]: oh you are right, should be boolean :(

**Pull request merged**